governance/polls/Base Rate Adjustment - August 03, 2020.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Base Rate Adjustment - August 03, 2020
+summary: Signal your support to adjust the Base Rate within a range of -4% to 4%
+discussion_link: https://forum.makerdao.com/t/3318
+vote_type: Ranked Choice IRV
+options:
+   0: Abstain
+   1: -4
+   2: -2
+   3: -1
+   4: -0.5
+   5: -0.25
+   6: 0
+   7: 0.25
+   8: 0.5
+   9: 1
+   10: 2
+   11: 4
+---
+# Poll: Base Rate Adjustment - August 03, 2020
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) which presents several possible **Base Rate** options. Voters are now able to signal their support for a **Base Rate** within a range of-4% to 4%
```

Comment:
as I guess this a very common misunderstanding about the negative baserate, i would put it as topmost as possible and as bold, blinking and maybe even use `<marquee>` for it stating that a negative base rate is *not* leading to a negative stability for any asset as there is a lower boundary of 0 for every stability fee on any collateral

i am pretty bad with words, so no real suggestion from my side ;-)

Reply:
Agree with this. Here is some text:

Stability Fees are calculated from the Base Rate and Risk Premiums using the formula `Stability Fee = max(Base Rate + Risk Premium, 0%)`. Please note that a negative **Base Rate** does **NOT** imply negative **Stability Fees**.

Reply:
Great, thanks guys, I'll add this in tomorrow.